watertap/tools/oli/watertap_to_oli_helper_functions.py

```diff
@@ -0,0 +1,195 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+#################################################################################
+
+"""
+This file contains methods to convert WaterTAP naming conventions to OLI
+and generate molecular weight and charge dictionaries from molecular formulae. 
+
+It calculates molecular weights using the periodic_table.csv from:
+https://gist.github.com/GoodmanSciences/c2dd862cd38f21b0ad36b8f96b4bf1ee.
```

Comment:
Does this satisfy any request from the source that might exist for citing?

Reply:
I'm unable to find any citation requirements